test-packages/e2e-tests/test/mail/mail.spec.ts

```diff
@@ -0,0 +1,41 @@
+import nodemailer from 'nodemailer';
+import fs from 'fs';
+import {join, resolve} from "path";
+import SMTPTransport from "nodemailer/lib/smtp-transport";
+
+describe('Mail', () => {
+  it('should send a mail', async () => {
+    await sendTestMail();
+    const mail = fs.readFileSync(join(resolve('test'), 'mail', 'test-output', 'mail.txt'),
+      { encoding: 'utf8'});
+    expect(mail).toContain('To: TO1@example.com, TO2@example.com');
+    expect(mail).toContain('Subject: SUBJECT');
+    expect(mail).toContain('TEXT');
+  });
+});
+
+async function sendTestMail(): Promise<SMTPTransport.SentMessageInfo>{
+  // create reusable transporter object using the default SMTP transport
+  let transporter = nodemailer.createTransport({
+    host: "localhost",
+    port: 5566,
```

Comment:
are these ports 5566, 465 a fixed value or do they are configured by the mail server you set up in `mail-server.js`?

Reply:
Ports can be configured.